bucket/wyrmsun.json

```
@@ -1,21 +1,20 @@
 {
-    "version": "3.5.4",
-    "description": "Real-time strategy game with retro graphics",
     "homepage": "https://andrettin.github.io/",
+    "description": "Real-time strategy game with retro graphics. It's also on Steam.",
```

I feel Steam presence is off-topic for the description; many other games packaged here are on Steam, but we don't denote it for every game.

```suggestion
    "description": "Real-time strategy game with retro graphics",
```